ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SketchOnMap/SketchOnMap.qml

```diff
@@ -0,0 +1,374 @@
+// [WriteFile Name=SketchOnMap, Category=DisplayInformation]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.15
+import Esri.Samples 1.0
+
+Item {
+    SketchOnMapSample {
+        id: model
+        mapView: view
+    }
+
+    MapView {
+        id: view
+        anchors.fill: parent
+    }
+
+    Control {
+        id: control
+        width: 110
+        anchors.right: parent.right
+        padding: 5
+
+        background: Rectangle {
+            color: "black"
+            opacity: .5
+        }
+
+        contentItem: ColumnLayout {
+            id: columns
+            anchors.verticalCenter: parent.verticalCenter
+            anchors.horizontalCenter: parent.horizontalCenter
+            spacing: 20
+
+            GridLayout {
+                id: geometryColumn
+                Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                columns: 2
+
+                Text {
+                    id: geometryText
+                    color: "#ffffff"
+                    text: qsTr("Geometry")
+                    font.pixelSize: 16
+                    Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                    font.bold: true
+                    Layout.columnSpan: 2
+                }
+
+                RoundButton {
+                    id: ptButton
+                    radius: 5
+                    Layout.fillWidth: true
+                    Layout.alignment: Qt.AlignHCenter
+
+                    Image {
+                        id: ptImage
+                        source: "iconAssets/point-32.png"
+                        anchors {
```

Comment:
usually as a matter of convention, we order QML properties in the following order:

1. ID (if there is one)
2. anchors or XY (if there are)
3. all other properties. 

I think this is written down somewhere, but not certain